### PR TITLE
HCI fixes #83

### LINKED ISSUE #83: Write interaction design report

### PR DESCRIPTION
**Pull request by eanker**
The part of the usability needs to be expaned after the session.

### CODE REVIEW

File: Documentation/General/finalReport/InteractionDesign.tex [hunk 1]
```diff
@@ -1,2 +1,106 @@
-\chapter{Human-Computer Interaction }
+\chapter{Human-Computer Interaction}
+
+\section{Introduction}
+This section describes the interaction between the user and the computer and how we measured this.
```
Comment (jwgmeligmeyling) [inline Documentation/General/finalReport/InteractionDesign.tex]: between the user and Moodcat sounds better to me.

File: Documentation/General/finalReport/InteractionDesign.tex [hunk 2]
```diff
@@ -1,2 +1,106 @@
...
+The user is very important for us, the user is the one that should use the application.
```
Comment (jwgmeligmeyling) [inline Documentation/General/finalReport/InteractionDesign.tex]: should -> will

File: Documentation/General/finalReport/InteractionDesign.tex [hunk 3]
```diff
@@ -1,2 +1,106 @@
...
+Our system is build for interaction between users and is improving by user interaction.
```
Comment (jwgmeligmeyling) [inline Documentation/General/finalReport/InteractionDesign.tex]: > and ~~is improving by~~ **the system is even learning** from user interaction.

File: Documentation/General/finalReport/InteractionDesign.tex [hunk 4]
```diff
@@ -1,2 +1,106 @@
...
+The interaction between users is the chatroom that our whole music application is build around.
```
Comment (jwgmeligmeyling) [inline Documentation/General/finalReport/InteractionDesign.tex]: > The interaction between users is the chatroom that our ~~whole~~ music application is build around.

File: Documentation/General/finalReport/InteractionDesign.tex [hunk 5]
```diff
@@ -1,2 +1,106 @@
...
+The music itself is based on valence and arousal witch is classified by the user.
```
Comment (jwgmeligmeyling) [inline Documentation/General/finalReport/InteractionDesign.tex]: I would not go into this detail for the HCI chapter. Just note that the music suggestions are based on the current / specified mood of the user.

File: Documentation/General/finalReport/InteractionDesign.tex [hunk 6]
```diff
@@ -1,2 +1,106 @@
...
+So, the more users our system has the better it will work.
```
Comment (jwgmeligmeyling) [inline Documentation/General/finalReport/InteractionDesign.tex]: This sentence can only follow after an argument why the system learns from the user, but this argument seems to be missing. Maybe remove this sentence?

File: Documentation/General/finalReport/InteractionDesign.tex [hunk 7]
```diff
@@ -1,2 +1,106 @@
...
+
+%• Gebruikersverhalen: duidelijk maken wat mensen willen en doen (0)
+%• Conceptuele scenario's: genereren van ideeën en vaststellen van requirements (0)
+%• Concrete scenario’s: Gebruik maken van prototypes en voor evaluatie (1)
+%• Use cases: voor documentatie en implementatie van interactie (3)
+
+\section{The user and their situation}
+
+\subsection{Use case scenarios}
+Here are a few use case scenarios, we have made more.
```
Comment (jwgmeligmeyling) [inline Documentation/General/finalReport/InteractionDesign.tex]: Obviously we thought of more use cases, but we have not documented them. Furthermore, I would not say: "Here are a few use case scenarios."
I would suggest to remove this sentence, and say something like:

> In this chapter we will elaborate the use cases we have made for our product.

File: Documentation/General/finalReport/InteractionDesign.tex [hunk 8]
```diff
@@ -1,2 +1,106 @@
...
+Due to readability we decided to only write down the 3 most important use case scenarios.
```
Comment (jwgmeligmeyling) [inline Documentation/General/finalReport/InteractionDesign.tex]: Then you might want to remove this as well.

File: Documentation/General/finalReport/InteractionDesign.tex [hunk 9]
```diff
@@ -1,2 +1,106 @@
...
+
+\subsubsection{User wants to play music on MoodCat.me.}
+\begin{itemize}
+\item The user goes to moodcat.me in his browser
```
Comment (jwgmeligmeyling) [inline Documentation/General/finalReport/InteractionDesign.tex:24]: we should maybe note that moodcat is a webpage music service.

File: Documentation/General/finalReport/InteractionDesign.tex [hunk 10]
```diff
@@ -1,2 +1,106 @@
...
+\item The user clicks on the mood he is in
+\item The user clicks next
+\item The user selects a room
+\end{itemize}
+
+\subsubsection{Whilst in a room the user wants to chat}
```
Comment (jwgmeligmeyling) [inline Documentation/General/finalReport/InteractionDesign.tex:30]: Not sure if im mistaken, but typo?

File: Documentation/General/finalReport/InteractionDesign.tex [hunk 11]
```diff
@@ -1,2 +1,106 @@
...
+\item User clicks submit
+\item User gets 6 points
+\item User comes back to the menu to click a new song to classify
+\end{itemize}
+
+
+%Beschrijving van context inquiry
+\section{Context inquiry}
+% A contextual inquiry interview is usually structured as an approximately two-hour, one-on-one interaction in which the researcher watches the user do their normal activities and discusses what they see with the user.
+We did a context inquiry with 5 other students a the tu delft.
```
Comment (jwgmeligmeyling) [inline Documentation/General/finalReport/InteractionDesign.tex]: > ~~tu delft~~ **TU Delft**.

File: Documentation/General/finalReport/InteractionDesign.tex [hunk 12]
```diff
@@ -1,2 +1,106 @@
...
+All 5 of us watched a different person, but we were sitting in the same room.
```
Comment (jwgmeligmeyling) [inline Documentation/General/finalReport/InteractionDesign.tex]: > All 5 of us watched a different person,

Why is this relevant?

>  ~~but we were sitting in the same room.~~ The students joined the same chat room.

Separate sentence (or put this first).

File: Documentation/General/finalReport/InteractionDesign.tex [hunk 13]
```diff
@@ -1,2 +1,106 @@
...
+They were separated by some free space, so they would not be influenced by each other.
+We just asked them to play with the system and we would see how they do.
+To see how they experienced the system is quite interesting, a user acts different in the first 5 minutes of system use.
```
Comment (jwgmeligmeyling) [inline Documentation/General/finalReport/InteractionDesign.tex]: We have a few times `5` now.

File: Documentation/General/finalReport/InteractionDesign.tex [hunk 14]
```diff
@@ -1,2 +1,106 @@
...
+%More about context inquiry
+
+%Beschrijving van product design, claim analyze over gebruik ondersteuned met HCI literatuur
+\section{Product design and claim analysis}
+MoodCat is developed with the end-user as starting point, we tried to make the interface as intuitive as possible.
+The interface should not contain a lot of information, so that the user remains the overview.
+%What to say more
+
+\subsection{Claim analysis}
+MoodCat is developed, so that people can listen to music and chat with people who are in the same mood.
```
Comment (jwgmeligmeyling) [inline Documentation/General/finalReport/InteractionDesign.tex]: The idea of Moodcat is to enjoy music together.

File: Documentation/General/finalReport/InteractionDesign.tex [hunk 15]
```diff
@@ -1,2 +1,106 @@
...
+We have resourced this and gamers streaming on twitch is very popular, due to the twitch chat.
```
Comment (jwgmeligmeyling) [inline Documentation/General/finalReport/InteractionDesign.tex]: Twitch

File: Documentation/General/finalReport/InteractionDesign.tex [hunk 16]
```diff
@@ -1,2 +1,106 @@
...
+A record company Monstercat streams music on twitch, this is very popular and people seem to enjoy this.
```
Comment (jwgmeligmeyling) [inline Documentation/General/finalReport/InteractionDesign.tex]: We don't like them.

File: Documentation/General/finalReport/InteractionDesign.tex [hunk 17]
```diff
@@ -1,2 +1,106 @@
...
+
+
+%Beschrijving van een bruikbaarheidsevaluatie uitgevoerd met het product
+\section{Usability evaluation}
+\subsection{Cognitive Walkthroughs}
+During the design of the product we did multiple usability evaluations.
+We have done quite a few Cognitive Walkthroughs, with changing possible end-users.
+We done this with the thinking out loud protocol, so we always knew there they where stuck or did not fully understand something.
+When you do not ask people to think out loud, they would sometimes hide some mistakes, which later on can be very important.
+We gave the users the task to explore the functionalities of the system.
```
Comment (jwgmeligmeyling) [inline Documentation/General/finalReport/InteractionDesign.tex:88]: I think we lack a detailed report on what actually were their experiences, what was the feedback that they came with, and did they use the system in ways we would have expected.

File: Documentation/General/finalReport/InteractionDesign.tex [hunk 18]
```diff
@@ -1,2 +1,106 @@
...
+We got some feedback and found that very valuable.
+Mainly, because in an early stage you know what goes wrong and right. 
+So, anticipating on the results is therefore a lot easier than later on.
+\subsection{GOMS}
+At the end of our project we did a GOMS analysis.
+We asked them to do some task and watched how quickly they preformed these tasks.
+These where the tasks of the Cognitive Walktrough, only split into multiple sections.
```
Comment (jwgmeligmeyling) [inline Documentation/General/finalReport/InteractionDesign.tex:95]: Again, I think we lack a detailed report on what actually were their experiences, what was the feedback that they came with, and did they use the system in ways we would have expected.

File: Documentation/General/finalReport/InteractionDesign.tex (hunk 10 above)
Comment (GijsWeterings) [inline Documentation/General/finalReport/InteractionDesign.tex:30]: no it's right

**Comment by jwgmeligmeyling** [comment]
Merging now...

**Pull request merged**